product_default_image/models/product_template.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import api, fields, models, tools
+
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class ProductTemplate(models.Model):
+
+    _inherit = 'product.template'
+
+    @api.model
+    def default_get(self, vals):
+        vals = super(ProductTemplate, self).default_get(vals)
+
+        if vals.get('categ_id'):
+
+            company = self.env.user.company_id
```

Comment:
So I functionally tested the module in runbot and it works. However with the tests, I'm having an issue with this line here.

If I change self.env.user.company_id in the tests, logs show in the test environment the company is correctly changed, however this company here is still left unchanged.

Was wondering if someone could point out how I could properly do that.

Thanks!